### PR TITLE
HPCC-17929 Change structure for ESDLLangRef in CHM

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>

@g-pan Please review.  Complied PDFs and Help are on Jenkins server

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/DynamicESDL/DESDL-Mods/ESDLbooleanbool.xml [hunk 1]
```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
 "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
-<sect2 id="boolean_bool">
-  <title><emphasis role="bold">boolean bool</emphasis></title>
+<sect1 id="boolean_bool">
```
Comment (g-pan) [inline docs/DynamicESDL/DESDL-Mods/ESDLbooleanbool.xml]: please change "boolean_bool" to ESDL_boolean or ESDL_boolean_bool for consistency with other id attributes.

File: docs/DynamicESDL/ESDL_LangRef_Includer.xml [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    <xi:include href="DynamicESDL/DESDL-Mods/ESDLESPservice.xml"
+                xpointer="element(/1)"
+                xmlns:xi="http://www.w3.org/2001/XInclude" />
+
+    <xi:include href="DynamicESDL/DESDL-Mods/ESDLESPmethod.xml"
+                xpointer="element(/1)"
+                xmlns:xi="http://www.w3.org/2001/XInclude" />
+  </chapter>
+
+  <chapter>
```
Comment (g-pan) [inline docs/DynamicESDL/ESDL_LangRef_Includer.xml]: please add id attribute (id=ESDL_Datatypes) to chapter

**Review by g-pan — COMMENTED** [review]
Couple minor comments.

**Comment by JamesDeFabia** [comment]
@g-pan requested changes made, back to you

**Comment by g-pan** [comment]
Reviewed sources and build results. Approved.

**Comment by JamesDeFabia** [comment]
@richardkchapman This is an important fix that I want to make sure is merged into 6.4 before gold. 
No content changes, just mechanical. Has been tested and reviewed.

**Pull request merged**